gemini-vertex-model-provider/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/obot-platform/tools/gemini-vertex-model-provider/server"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/genai"
+)
+
+func main() {
+	ctx := context.Background()
+
+	args := os.Args[1:]
+	if len(args) == 1 && args[0] == "validate" {
+		if err := validate(ctx); err != nil {
+			fmt.Printf("{\"error\": \"%s\"}\n", err)
+		}
+		os.Exit(1)
```

Comment:
We probably don't want to always `exit 1` for `validate.

Reply:
Probably